Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +38,17 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0)
+            {
+                return;
+            }
+
+            for (int i = 0; i < array.Length / 2; i++)
+            {
+                int temp = array[i];
+                array[i] = array[array.Length - i - 1];
+                array[array.Length - i - 1] = temp;
```

Comment:
I guess Swap() would work here, too?

Reply:
Thanks for pointing it out, I forgot to replace the code.

`SwapElements(array, i, array.Length - i - 1);`